scripts/hail_batch/hgdp1kg_tobwgs_pca/hgdp_1kg_tob_wgs_pca.py

```diff
@@ -0,0 +1,98 @@
+"""Generates loadings, scores, and eigenvalues for the HGDP,1KG, and tob-wgs dataset"""
+
+import click
+import pandas as pd
+import hail as hl
+from hail.experimental import lgt_to_gt
+
+GNOMAD_HGDP_1KG_MT = (
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
+    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+)
+
+TOB_WGS = 'gs://cpg-tob-wgs-temporary/joint_vcf/v1/work/combiner/genomes.mt'
+
+GNOMAD_V2_LOADINGS = (
+    'gs://gcp-public-data--gnomad/release/2.1/pca/gnomad.r2.1.pca_loadings.ht'
+)
+
+
+@click.command()
+@click.option('--output', help='GCS output path', required=True)
+def query(output):  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    # Filter the matrix tables to rows whose keys appear in both datasets
+    mt_hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
+    mt_tob_wgs = hl.read_matrix_table(TOB_WGS)
+    # Remove 'allele' row key so that rows are the same (necessary step to join)
+    mt_hgdp_1kg = mt_hgdp_1kg.key_rows_by('locus')
+    mt_hgdp_1kg = mt_hgdp_1kg.drop('alleles')
```

Comment:
@vladsaveliev Do we have a convenience function to move a sparse MT to the typical `locus` + `alleles`-keyed table format?

The code below is a bit tricky to follow (e.g. dropping `alleles` seems odd) and I'd expect such functionality to be useful occasionally.

Reply:
A table can be rekeyed by locus/alleles with:

```
mt_tob_wgs = mt_tob_wgs.key_rows_by('locus', 'alleles')
```

Can modify the combiner to do that automatically.

Reply:
thanks Vlad, I've changed it to `mt_tob_wgs = mt_tob_wgs.key_rows_by('locus', 'alleles')`. Looks a lot cleaner

Reply:
@vladsaveliev Is the new MT in `main` already keyed this way?

Reply:
It's not, but in any case, it's safe to leave that line in the code just for a more robust code. I didn't find it to add any overhead if the MT is already keyed by locus+allele.